node/cmd/guardiand/node.go

```diff
@@ -961,6 +961,15 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger = tm.WrapLogger(logger)
 	} else {
 		logger.Info("Telemetry disabled")
+
+		externalLogger := &telemetry.ExternalLoggerNil{}
```

Comment:
given you're passing in a pointer anyway I think just accepting nil as input would make sense.

Reply:
For tests, a "testExternalLogger" could be used that sits in the test file.

Reply:
I wanted to include this `ExternalLoggerNil` stub logger in devmode such that the code paths involving the cloning of the logger etc. are covered in integration tests. That's why it's not just local to the test file. 
Wdyt?

Reply:
I'd prefer accepting nil and just writing a unit test that exercises the clone.

Reply:
Ok. Reverted changes to `node.go` and made `externalLoggerMock` local to `telemetry_test.go`